Extension/package.json

```diff
@@ -3376,6 +3376,12 @@
                         "default": "default",
                         "markdownDescription": "%c_cpp.configuration.copilotHover.markdownDescription%",
                         "scope": "window"
+                    },
+                    "C_Cpp.persistDevEnvironment": {
+                        "type": "boolean",
+                        "default": true,
+                        "markdownDescription": "%c_cpp.configuration.persistDevEnvironment.markdownDescription%",
+                        "scope": "resource"
```

Comment:
This should be "window" scope, right? i.e. per-workspace and not per-workspace folder.

Reply:
yes, I think you are right.